python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -2091,20 +2089,19 @@ def __setattr__(self, attr, value):
 
             # reduce duplicate values here, leave only duplicates for LCG,
             # where we can have replicas
-            uniqueValuesDict = []
-            uniqueValues = []
+            uniqueValuesDict = oDict()
```

Comment:
I was looking to reduct the total number of objects created and the special test for LCGSEFile (there were lots of lists created which didn't look appropriate. I'll switch back to lists which is imo more confusing but is what we had, and this shouldn't effect the user as in both cases they just get a list back)